routes/users/search.js

```diff
@@ -18,7 +18,7 @@ const findUser = (result, accessToken) => new Promise((resolve) => {
   }
 
   // Otherwise, search for it
-  return orcid.getPersonDetails(userOrcID, accessToken, (userErr, userData) => {
+  return orcid.getPersonDetails(userOrcID, null, (userErr, userData) => {
```

Comment:
Any specific the purpose for skipping the token?
Bypassing token-linked rate limits?

Reply:
@alexandru-patrascu-wingravity You can take the access token from `req`. Check this as an example: https://github.com/ScienceOctopus/octopus-ui/pull/4/files#diff-0c6c3ba4203c49b739f706f1fd4f405fR71